src/modules/launcher/Plugins/Microsoft.Plugin.Program/SearchResult.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Plugin.Program.Programs;
+
+namespace Microsoft.Plugin.Program
+{
+    internal class SearchResult
```

Comment:
I think it would be better to make `ProgramArguments` part of `Result` class. `SearchResult` here seems to have the same functionality as `Result`. @ryanbodrug-microsoft Thoughts ?

Reply:
My reasoning for not doing that was I did not want to change any of the wox namespaces, it was my fist idea as well. But it is something to consider.

Reply:
@royvou It is fine to make changes in `wox` namespace. And I feel this would be a cleaner way to do this.

Reply:
Will update the PR later this week !

Reply:
Great!! Thanks.

Reply:
Just a note the PR is updated 👍